src/cuda/symbols.def

```diff
@@ -0,0 +1,2 @@
+EXPORTS
```

Comment:
Could we use `__declspec(dllexport)` instead similar to how we handle the ort-genai C API?

Reply:
It won't work across all platforms, that's why I switched to using the separate file in cmake.